pkg/backend/httpstate/backend.go

```diff
@@ -594,7 +631,7 @@ func (b *cloudBackend) Query(ctx context.Context, stackRef backend.StackReferenc
 
 func (b *cloudBackend) createAndStartUpdate(
 	ctx context.Context, action apitype.UpdateKind, stack backend.Stack,
-	op backend.UpdateOperation, dryRun bool) (client.UpdateIdentifier, int, string, error) {
+	op *backend.UpdateOperation, dryRun bool) (client.UpdateIdentifier, int, string, error) {
```

Comment:
No need for this to be a pointer?

Reply:
We're mutating `op` below, you can see a large comment why we're doing this and our plan to change this when we have a dedicated API for listing the required policies associated with some update.

Reply:
Okay, that makes sense. Gross.